### ISSUE TITLE
Losing points due to too large changelog

### ISSUE THREAD

**Issue by spydon** [dart-lang/pub-dev#7118]
![image](https://github.com/dart-lang/pub-dev/assets/744771/441c746e-889f-414c-ab73-8f43973d2f85)

In Flame we have an exceptionally good [changelog](https://github.com/flame-engine/flame/blob/main/packages/flame/CHANGELOG.md), I think it sends the wrong signals from Pub to get minus points for this.
(Although we are very happy that we can now publish without modifications after the size limit was [bumped to 256KB](https://github.com/dart-lang/pub-dev/pull/7065)! 🥳)

If there is a technical issue by pub not being able to handle the file size, I think there should be a proper solution for this and something that should be solved on Pub's side, not on the package author's side.

Related: #6012

Any new users searching for how to keep a good changelog will get results like these in the top:
https://keepachangelog.com/
https://docs.gitlab.com/ee/development/changelog.html
https://github.com/olivierlacan/keep-a-changelog/blob/main/CHANGELOG.md
https://www.freecodecamp.org/news/a-beginners-guide-to-git-what-is-a-changelog-and-how-to-generate-it/
None which recommend splitting up the changelog into multiple files, keeping all changes in `CHANGELOG.md` has been the de facto open source standard for quite a while.

**Comment by renancaraujo** [comment]
if the biggest technical problem with large change logs is page load, the site could truncate the content up until a certain size.

**Comment by rydmike** [comment]
Ok this is just silly, now you loose pub points for being thorough and documenting things well with a good detailed changelog!?

This sends the wrong signal to devs. Encouraging logs like "We fixed a number of bugs". Logs that are not very useful, but will keep it short to stay below the 128kB limit for getting the desired "valid" changelog points.

**Comment by spydon** [comment]
Maybe the issue with points being deducted might be accidental, that this line just wasn't updated together with #7065:
https://github.com/dart-lang/pana/blob/master/lib/src/report/template.dart#L26

**Comment by rydmike** [comment]
@spydon looks likely, since it would be really easy to forget that line and check when PR https://github.com/dart-lang/pub-dev/pull/7065 was done. This limit default does not depend on any of the constants used for the upload sizes, it is just a hard coded number default. If call site then just uses the defaults, then bam, what we got now would be the result.

**Comment by sigurdm** [comment]
Yeah, this is just an accidental oversight - @isoos we should also fix this in pana!

**Comment by isoos** [comment]
Ah, good point, sorry for the confusion it made! Transferring the issue to pana.
